packages/example-app-base/src/components/SideRail/SideRail.types.ts

```diff
@@ -0,0 +1,50 @@
+import { ITheme, IStyleFunctionOrObject, IStyle } from 'office-ui-fabric-react';
+
+export interface ISideRailProps {
+  /**
+   * List of hash links to places within the current page.
+   */
+  jumpLinks?: ISideRailLink[];
+
+  /**
+   * List of links to related pages, as link objects or pre-rendered.
+   */
+  relatedLinks?: ISideRailLink[] | JSX.Element;
+
+  /**
+   * Mailto links for people related to the page, as link objects or pre-rendered.
+   */
+  contactLinks?: ISideRailLink[] | JSX.Element;
+
+  /**
+   * If the page has elements to observe.
+   */
+  observe?: boolean;
+
+  /**
+   * Override styles.
+   */
+  styles?: IStyleFunctionOrObject<ISideRailStyleProps, ISideRailStyles>;
+
+  /**
+   * Theme provided by higher-order component.
+   */
+  theme?: ITheme;
+}
+
+export interface ISideRailLink {
+  text: string;
+  url: string;
```

Comment:
nit: Will we have URLs or paths? Based on context, I suspect `path` is a more likely name for this value.

Reply:
URL is accurate since we can have links to internal content or external links (mailto, 3rd party related content, etc).

Reply:
Added comments :)